orttraining/orttraining/python/training/ortmodule/_custom_autograd_function_exporter.py

```diff
@@ -165,6 +165,14 @@ def _export(g, n, *args, **kwargs):
         raise wrap_exception(ORTModuleONNXModelException, e)
 
 
+try:
+    from torch.onnx import SymbolicContext
+    def _export(g, ctx: SymbolicContext, *args, **kwargs):
+        n = ctx.cur_node
+        return _export_pt_1_10(g, n, *args, **kwargs)
+except:
```

Comment:
If you expect one specific exception, you should use it. No function `_export_pt_1_9`?

Reply:
updated.